src/schema/schema.graphql

```diff
@@ -1371,6 +1377,11 @@ type ContactListMember {
 	value: ID!
 }
 
+input otherInputAttribute {
```

Comment:
OtherContactAttributeInput

Reply:
ContactExtraAttributeInput.. Why do we call them other when they are just extra?

Reply:
It was mentioned in the ticket that all the remaining values will be called other, hence added it, Please let me know if I need to rename it.